utils/config.go

```diff
@@ -46,6 +47,25 @@ var ClientCfg map[string]string = map[string]string{}
 var originalDisableDebugLvl l4g.Level = l4g.DEBUG
 var siteURL = ""
 
+var awsS3EndpointMap = map[string]string{
+	"s3.amazonaws.com":                "us-east-1",
+	"s3-us-east-2.amazonaws.com":      "us-east-2",
+	"s3-us-west-2.amazonaws.com":      "us-west-2",
+	"s3-us-west-1.amazonaws.com":      "us-west-1",
+	"s3.ca-central-1.amazonaws.com":   "ca-central-1",
+	"s3-eu-west-1.amazonaws.com":      "eu-west-1",
+	"s3-eu-west-2.amazonaws.com":      "eu-west-2",
+	"s3-eu-central-1.amazonaws.com":   "eu-central-1",
+	"s3-ap-south-1.amazonaws.com":     "ap-south-1",
+	"s3-ap-southeast-1.amazonaws.com": "ap-southeast-1",
+	"s3-ap-southeast-2.amazonaws.com": "ap-southeast-2",
+	"s3-ap-northeast-1.amazonaws.com": "ap-northeast-1",
+	"s3-ap-northeast-2.amazonaws.com": "ap-northeast-2",
+	"s3-sa-east-1.amazonaws.com":      "sa-east-1",
+	"s3-us-gov-west-1.amazonaws.com":  "us-gov-west-1",
+	"s3.cn-north-1.amazonaws.com.cn":  "cn-north-1",
+}
```

Comment:
Any way we can use [aws/endpoints](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/) or something a bit more authoritative rather then maintaining this map?

Reply:
Just copied from here (and modified a bit) - https://github.com/minio/minio-go/blob/master/s3-endpoints.go. If that's expose then it should be convenient to use.